[2025-02-05T06:50:19.648+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:50:19.696+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: task_group_example.end_task scheduled__2024-02-17T00:00:00+00:00 [queued]>
[2025-02-05T06:50:19.699+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: task_group_example.end_task scheduled__2024-02-17T00:00:00+00:00 [queued]>
[2025-02-05T06:50:19.700+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2025-02-05T06:50:19.703+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): end_task> on 2024-02-17 00:00:00+00:00
[2025-02-05T06:50:19.705+0000] {standard_task_runner.py:57} INFO - Started process 3324 to run task
[2025-02-05T06:50:19.707+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'task_group_example', 'end_task', 'scheduled__2024-02-17T00:00:00+00:00', '--job-id', '350', '--raw', '--subdir', 'DAGS_FOLDER/task_group_dag.py', '--cfg-path', '/tmp/tmpkiewsq4w']
[2025-02-05T06:50:19.707+0000] {standard_task_runner.py:85} INFO - Job 350: Subtask end_task
[2025-02-05T06:50:19.713+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:50:19.724+0000] {task_command.py:415} INFO - Running <TaskInstance: task_group_example.end_task scheduled__2024-02-17T00:00:00+00:00 [running]> on host 3849bfb82152
[2025-02-05T06:50:19.728+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:50:19.751+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='task_group_example' AIRFLOW_CTX_TASK_ID='end_task' AIRFLOW_CTX_EXECUTION_DATE='2024-02-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-17T00:00:00+00:00'
[2025-02-05T06:50:19.752+0000] {logging_mixin.py:151} INFO - 执行任务: 结束任务
[2025-02-05T06:50:19.752+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-05T06:50:19.754+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=task_group_example, task_id=end_task, execution_date=20240217T000000, start_date=20250205T065019, end_date=20250205T065019
[2025-02-05T06:50:19.764+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-05T06:50:19.770+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
